magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/source_dict.json

```diff
@@ -0,0 +1,125 @@
+"PSRJ2032":[308.0833333333333, 41.51, "True"],
+"Mrk421-Mon":[166.11380868146, 38.20883291552, "True"],
+"Mrk421-Muons":[166.11380868146, 38.20883291552, "True"],
+"Mrk501-Mon":[253.467569516792, 39.760169132056, "True"],
+"M87-Mon":[187.70593076725, 12.391123246083, "True"],
+"GRB221218A":[350.6500, -4.25, "True"],
+"GRB231110":[356.29583333333335, 82.62083333333334, "True"],
+"GRB231111A":[290.1885, 52.43333, "True"],
+"GRB231215A":[9.741666666666667, 57.646388888888886, "True"],
+"GRB231216A":[39.7727, 33.56667, "True"],
+"GRB240502A":[113.6292, 16.71667, "True"],
+"GRB240529":[335.3208333333333, 51.555, "True"],
+"GRB210511B":[316.1791666666667, 61.69, "True"],
+"GRB220311A":[157.975, 66.08194444444445, "True"],
+"GRB240615A":[326.1417, 38.5833, "True"],
+"Boomerang-Ta":[336.75, 60.84, "False"],
+"BLLac":[330.6804167, 42.27777206022, "True"],
+"LSI+61":[40.13333333333333, 61.22944444444445, "True"],
+"S51803+784":[270.1916667, 78.46667, "True"],
+"S51044+71":[162.115, 71.71667, "True"],
+"S40954+65":[149.697, 65.5652275985, "True"],
+"3C-345.0":[250.745041, 39.810276098667, "True"],
+"B2-1420+32":[215.625, 32.38333, "True"],
+"B21348+30B":[207.71666666666667, 30.58138888888889, "True"],
+"FRB-20200120":[149.475, 68.81666666666666, "True"],
+"GalacticCent":[266.4166666666667, -29.00777777777778, "False"],
+"MAXIJ1848-01":[282.2, -1.4867, "NaN"],
+"1ES1028+51":[157.82718791991, 50.89328257724, "True"],
+"1A-0535+262":[84.72739538276, 26.31578804244, "NaN"],
+"Cyg-X3":[308.107417, 40.95, "True"],
+"S20109+22":[18.0241667, 22.73333, "True"],
+"S30218+35":[35.27279, 35.93333, "True"],
+"LHAASOJ2108+":[317.15000000000003, 51.95, "False"],
+"LhaasoJ1956+":[299.05, 28.75, "False"],
+"PSRJ2229+611":[337.271925, 61.2356889, "True"],
+"1LHAASOJ0007":[1.7541666666666667, 73.05194444444444, "True"],
+"4FGLJ1342.7+":[205.67916666666667, 5.075555555555556, "True"],
+"4FGLJ1723.5-":[260.96250000000003, -5.153888888888889, "NaN"],
+"3FHLJ0658.5+":[104.6875, 6.619722222222222, "True"],
+"HESS1857+026":[284.3625, 2.716666666666667, "False"],
+"LHAASOJ0341+":[55.3375, 52.97,"False"],
+"LHHASOJ0341+":[55.3375, 52.97, "False"],
+"SDSSJ1430+23":[217.56666666666666, 23.06222222222222, "NaN"],
+"GRB-230812B":[250.00000000000003, 46.2, "True"],
+"GRB-230812B-":[248.93750000000003, 47.945, "True"],
+"S240615dg":[7.520833333333333, 45.80888888888889, "NaN"],
+"IC20220624A":[224.11666666666667, 41.31, "NaN"],
+"IC-220303A":[267.79999999999995, 11.42, "NaN"],
+"RGB0521+211":[80.44152391208, 21.21429244076, "True"],
+"FermiJ0330.1":[52.50833333333333, 37.73, "NaN"],
+"HESSJ1912":[288.26666666666665, 10.207777777777778, "False"],
+"Abell-1132-M":[164.68333333333334, 56.592777777777776, "NaN"],
+"Cygnus-XNort":[310.0416666666667, 42.471944444444446, "NaN"],
+"GBR230512A":[295.39583333333337, 38.39972222222222, "True"],
+"GRB220603":[284.15, 18.84, "True"],
+"HESSJ1848-01":[282.12083333333334, -1.0527777777777778, "False"],
+"Perseus-MA":[49.5625, 41.418055555555554, "NaN"],
```

Comment:
this is not a real source but a position in between NGC 1275 and IC 310, so you can put "False" to point like